mypy/nodes.py

```diff
@@ -117,11 +116,17 @@ def accept(self, visitor: NodeVisitor[T]) -> T:
         raise RuntimeError('Not implemented')
 
 
-# These are placeholders for a future refactoring; see #1783.
-# For now they serve as (unchecked) documentation of what various
-# fields of Node subtypes are expected to contain.
-Statement = Node
-Expression = Node
+class Expression(Node):
+    pass
+
+
+class Statement(Node):
+    pass
+
+
+# For now, only Var is a declaration
+class Declaration(Node):
+    pass
 
 
 class SymbolNode(Node):
```

Comment:
Honestly the first refactor that *I* would like to see is to make SymbolNode not derive from Node...